### DISCUSSION TITLE
Please move the "Close pull request" button farther from the "Add comment" button.

### DISCUSSION THREAD

**Discussion by batrick** [community/community#122850] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Please imagine how aggravating it is to miss the "Comment" button only to close the PR.

**Comment by egorkel-altexsoft** [comment]
Today I missed the comment button :)
![image](https://github.com/community/community/assets/95917278/4d69d3f2-6b1b-4203-aef5-f1f69f80b801)
I click on the comment field (1) then I move my pointer to the comment button (2). The pointer track passes through the close PR button.
Please move the close button to the left side far from comment one

**Comment by batrick** [comment]
Did it again: https://github.com/ceph/ceph/pull/59162#event-13863138830

PLEASE?!
